tiledb/sm/rest/rest_client.cc

```diff
@@ -511,6 +512,59 @@ Status RestClient::post_array_metadata_to_rest(
       stats_, url, serialization_type_, &serialized, &returned_data, cache_key);
 }
 
+std::vector<shared_ptr<const Enumeration>>
+RestClient::get_enumerations_from_rest(
```

Comment:
I try to prefix with the http method, so `post_enumerations_from_rest`. Its a bad name, feel free to ignore but its what we've done for the other functions.

Reply:
I ended up going with the `post_enumerations_from_rest`. I had initially avoided it since it doesn't read very well, but using `get_enumerations_from_rest` inadvertently breaks the pattern of HTTP verbs in function names since GET is a valid HTTP verb.